Department.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Main;
+
+use ArrayObject;
+
+class Department
+{
+    private string $name;
+    private array $workers;
```

Comment:
- added type hinting for workers property
- added type hinting for name parameter
```suggestion
    private ArrayObject $workers;

    public function __construct(string $name, array $list)
```